src/passkey_child/passkey_child_common.c

```diff
@@ -82,6 +83,8 @@ parse_arguments(int argc, const char *argv[], struct passkey_data *data)
          _("Register a passkey for a user"), NULL },
         {"authenticate", 0, POPT_ARG_NONE, NULL, 'a',
          _("Authenticate a user with a passkey"), NULL },
+        {"format-pem", 0, POPT_ARG_NONE, NULL, 'f',
+         _("Format public key in PEM format and print it"), NULL },
         {"username", 0, POPT_ARG_STRING, &data->shortname, 0,
          _("Shortname"), NULL },
         {"domain", 0, POPT_ARG_STRING, &data->domain, 0,
```

Comment:
Btw, why does format-pem options have a description "Shortname"?

Reply:
I added a new commit fixing this point